### PR TITLE
refactor(itinerary-body): remove direct import of fa-solid

### PR DESCRIPTION
**Pull request by daniel-heppner-ibigroup**
This should fix the problem of the entirety of fa-solid being loaded into the otp-rr bundle.

### CODE REVIEW

**Comment by daniel-heppner-ibigroup** [comment]
I'm not sure why the yarn.lock changed, but I did use it as the fix commit. Is there something else to investigate there?

File: yarn.lock [hunk 1]
```diff
@@ -2832,6 +2832,14 @@
   resolved "https://registry.yarnpkg.com/@open-draft/until/-/until-1.0.3.tgz#db9cc719191a62e7d9200f6e7bab21c5b848adca"
   integrity sha512-Aq58f5HiWdyDlFffbbSjAlv596h/cOnt2DO1w3DOC7OJ5EHs0hd/nycJfiu9RJbT6Yk6F1knnRRXNSpxoIVZ9Q==
 
+"@opentripplanner/icons@1.2.4":
```
Comment (binh-dam-ibigroup) [inline yarn.lock:2835]: You can remove the changes to this file from the PR. The underlying issue is handled by #444.

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM

File: yarn.lock (hunk 1 above)
Comment (daniel-heppner-ibigroup) [reply yarn.lock:2835]: Sounds good, I will revert it

**Review by miles-grant-ibigroup — APPROVED** [review]
Change looks good, thanks for finding that. Approved provided merging master into this before merging this into master (the revert commit makes me nervous -- it may delete the `yarn.lock` changes!)

**Pull request merged**